prime-router/metadata/fhir_mapping/hl7/segments/ORC/ServiceRequest.yml

```diff
@@ -0,0 +1,375 @@
+resourceType: ServiceRequest
+
+# Not Implementing
+# ORC.7 deprecated in the NIST spec
+# ORC.26 deprecated in the NIST spec
+# ORC.32 not mapped, not in NIST spec
+# ORC.34 not mapped, not in NIST spec
+# OBR.5 deprecated in NIST spec
+# OBR.6 deprecated in NIST spec
```

Comment:
Looks like this is mapped as `occurrenceDateTime`

Reply:
Yep and I actually did map it, but forgot to delete it from here